### PR TITLE
set language via url GET param

### PR DESCRIPTION
**Pull request by brucebinary**

### CODE REVIEW

File: app/index.jsx [hunk 1]
```diff
@@ -54,10 +54,11 @@ function getServerUrl() {
 const chartId = '1';
 const appId  = CIQ.localStorage.getItem('config.app_id') || 12812;
 const serverUrl = getServerUrl();
+const language = new URLSearchParams(window.location.search).get('l') || getLanguageStorage();
```
Comment (shayanbinary) [inline app/index.jsx]: @brucebinary I think `URLSearchParams` may not work on `Edge 15` and `Safari 9`, would you please check? if yes, what is your idea? change our support version or make custom function beside this one?

Comment (brucebinary) [reply app/index.jsx]: We're not supporting those right? [This is our browserlist setting](https://browserl.ist/?q=defaults%2C+not+op_mini+all%2C+not+ie+%3C%3D+11%2C+not+IE_Mob+%3C%3D+11%2C+not+Samsung+%3C%3D+4)

Comment (brucebinary) [reply app/index.jsx]: ok just talked with binary-static team. We need to support all edge versions.

**Comment by brucebinary** [comment]
@shayanbinary I've updated the PR to use a polyfill if it is not supported.

**Comment by shayanbinary** [comment]
@brucebinary I don't know if you fix this or not, but your test link has the problem on `Edge 16`

**Comment by brucebinary** [comment]
@shayanbinary man. It just fails without any errors. But Edge 17 works. Damn.

**Comment by brucebinary** [comment]
@shayanbinary omg. `URLSearchParams` is only added in edge 17. *_*

**Pull request merged**